### DISCUSSION TITLE
https://docs.okd.io/ down

### DISCUSSION THREAD

**Discussion by var-log-skynet** [okd-project/okd#1147] [General]
Hi! Is this temporary or did I miss something?

**Comment by LorbusChris** [comment]
I saw this too over the weekend, but it seems resolved now.

**Reply by jorx-dev** [reply]
Hello, today is down again. Yesterday evening (CET) was up and running.

**Comment by LorbusChris** [comment]
@JaimeMagiera do you know where is the site hosted again? Any idea what's going on here?

**Comment by binnes** [comment]
The site is now hosted on github pages, so github must have had a temporary issue.  It all seems OK now
